### PR TITLE
Throw exception instead of returning "0000000000"

### PR DESCRIPTION
**Pull request by nithia**

### CODE REVIEW

File: exercises/phone-number/src/test/kotlin/PhoneNumberTest.kt [hunk 1]
```diff
@@ -32,21 +33,19 @@ class PhoneNumberTest {
     }
 
     @Ignore
-    @Test
+    @Test(expected = IllegalArgumentException::class)
     fun invalidWhenOnly11Digits() {
-        val expectedNumber = "0000000000"
         val actualNumber = PhoneNumber("21234567890").number
```
Comment (sdavids13) [inline exercises/phone-number/src/test/kotlin/PhoneNumberTest.kt]: We should just call the constructor without the `.number` it is unclear where the exception is being thrown (it should be done at object creation as it is currently).

File: exercises/phone-number/src/test/kotlin/PhoneNumberTest.kt [hunk 2]
```diff
@@ -32,21 +33,19 @@ class PhoneNumberTest {
...
-        assertEquals(expectedNumber, actualNumber)
+        fail("IllegalArgumentException should have been thrown")
     }
 
     @Ignore
-    @Test
+    @Test(expected = IllegalArgumentException::class)
     fun invalidWhen9Digits() {
-        val expectedNumber = "0000000000"
         val actualNumber = PhoneNumber("123456789").number
```
Comment (sdavids13) [inline exercises/phone-number/src/test/kotlin/PhoneNumberTest.kt]: Same comment w/r/t the `.number` here as well.

File: exercises/phone-number/src/test/kotlin/PhoneNumberTest.kt (hunk 1 above)
Comment (nithia) [inline exercises/phone-number/src/test/kotlin/PhoneNumberTest.kt]: Indeed, you are right. I will fix this

**Comment by sdavids13** [comment]
Awesome, thanks.

**Pull request merged**